Autonomy/app/src/main/java/nasa_rmc/autonomy/ForwardingPrefix.java

```diff
@@ -0,0 +1,24 @@
+package nasa_rmc.autonomy;
+
+/**
+ * Created by atomlinson on 4/3/17.
+ */
+
+public enum ForwardingPrefix {
```

Comment:
Will do. I'm going to go through and add comments to everything once I get the structure completely done.